mlflow/utils/virtualenv.py

```diff
@@ -150,11 +150,13 @@ def _install_python(version, pyenv_root=None, capture_output=False):
 
 
 def _get_conda_env_file(mlmodel_file):
```

Comment:
Shall we also update https://github.com/mlflow/mlflow/blob/898260413e9d111055f20290f0b776bbccc2dc07/mlflow/utils/virtualenv.py#L172 to read the python env file from the model conf instead of assuming a hardcoded path?

Reply:
Can we also apply this change ^ to other locations throughout the codebase where we assume a hardcoded python env file path

Reply:
Sure!